src/main/java/de/rwth/idsg/steve/service/AuthTagService.java

```diff
@@ -0,0 +1,85 @@
+package de.rwth.idsg.steve.service;
+
+import static de.rwth.idsg.steve.utils.OcppTagActivityRecordUtils.isBlocked;
+import static de.rwth.idsg.steve.utils.OcppTagActivityRecordUtils.isExpired;
+import static de.rwth.idsg.steve.utils.OcppTagActivityRecordUtils.reachedLimitOfActiveTransactions;
+
+import de.rwth.idsg.steve.repository.OcppTagRepository;
+import de.rwth.idsg.steve.repository.SettingsRepository;
+import jooq.steve.db.tables.records.OcppTagActivityRecord;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import ocpp.cs._2015._10.AuthorizationStatus;
+import ocpp.cs._2015._10.IdTagInfo;
+import org.jetbrains.annotations.Nullable;
+import org.joda.time.DateTime;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AuthTagService {
+  private final OcppTagRepository ocppTagRepository;
+  private final SettingsRepository settingsRepository;
+
+  public IdTagInfo decideStatus(String idTag,
+                                boolean isStartTransactionReqContext,
+                                @Nullable String chargeBoxId,
+                                @Nullable Integer connectorId) {
+    OcppTagActivityRecord record = ocppTagRepository.getRecord(idTag);
```

Comment:
the project uses 4-spaces indentation

note to self: we should enforce this by checkstyle.